cluster/addons.go

```diff
@@ -58,6 +59,8 @@ type ingressOptions struct {
 	AlpineImage       string
 	IngressImage      string
 	IngressBackend    string
+	HttpPort          string
```

Comment:
If these options could be passed via extra args, why do we need them here?

Reply:
I think the idea was to make them first class citizens in cluster spec.